standard/classes.md

```diff
@@ -412,23 +413,32 @@ Each *type_parameter_constraints_clause* consists of the token `where`, followed
+A primary constraint can be a class type, the ***reference type constraint*** `class`, the ***value type constraint*** `struct`, or the ***unmanaged type constraint*** `unmanaged`.
+
+A secondary constraint can be a *type_parameter* or *interface_type*.
 
 The reference type constraint specifies that a type argument used for the type parameter shall be a reference type. All class types, interface types, delegate types, array types, and type parameters known to be a reference type (as defined below) satisfy this constraint.
 
 The value type constraint specifies that a type argument used for the type parameter shall be a non-nullable value type. All non-nullable struct types, enum types, and type parameters having the value type constraint satisfy this constraint. Note that although classified as a value type, a nullable value type ([§8.3.11](types.md#8311-nullable-value-types)) does not satisfy the value type constraint. A type parameter having the value type constraint shall not also have the *constructor_constraint*, although it may be used as a type argument for another type parameter with a *constructor_constraint*.
 
 > *Note*: The `System.Nullable<T>` type specifies the non-nullable value type constraint for `T`. Thus, recursively constructed types of the forms `T??` and `Nullable<Nullable<T>>` are prohibited. *end note*
 
-Pointer types are never allowed to be type arguments and are not considered to satisfy either the reference type or value type constraints.
+The token `unmanaged` is neither a keyword nor a contextual keyword. When it is encountered, it will either:
+
+- Bind to a type named `unmanaged`
+- Bind to no type, in which case, it is interpreted as the unmanaged type constraint.
+
+The unmanaged type constraint specifies that a type argument used for the type parameter shall be a non-nullable unmanaged type.
+
+Pointer types are never allowed to be type arguments and are not considered to satisfy either the `reference` type or `value` type constraints.
 
 If a constraint is a class type, an interface type, or a type parameter, that type specifies a minimal “base type” that every type argument used for that type parameter shall support. Whenever a constructed type or generic method is used, the type argument is checked against the constraints on the type parameter at compile-time. The type argument supplied shall satisfy the conditions described in [§8.4.5](types.md#845-satisfying-constraints).
 
 A *class_type* constraint shall satisfy the following rules:
 
 - The type shall be a class type.
 - The type shall not be `sealed`.
-- The type shall not be one of the following types: `System.Array`, `System.Delegate`, `System.Enum`, or `System.ValueType`.
+- The type shall not be one of the following types: `System.Array` or `System.ValueType`.
```

Comment:
Do we highlight anywhere that `System.Delegate` and `System.Enum` are now allowed (as of v7.3)? Should we? (It seems a notable change from v<7.3)

Reply:
It would be good to include it in a list of changes since v6, but I don't think the relaxation warrants any other highlighting.

Reply:
On the TG2 call of 2023-02-02, Rex will add this to th list of "what's new in V7."

Reply:
Done!